homework.js

```diff
@@ -0,0 +1,108 @@
+//  1.
+//  sum all numbers, from 1 to num
+const summation = function (num) {
+    let sum = 0;
+    for (i = 1; i <= num; i++) {
+        sum += i;
+    }
+    return sum;
+}
+
+//  2.
+//  sum all even numbers equal or less than num
+const summationEven = function (num) {
+    let sum = 0;
+    if (num % 2 == 1)
+        num--;
+
+    return summation(num / 2) * 2;
+}
+
+//  3.
+//  average numbers array
+const avg = function (numArr) {
+    let sum = 0;
+    for (i = 0; i < numArr.length; i++) {
+        sum += numArr[i];
+    }
+    return sum / numArr.length;
+}
+
+//  4.
+//  reverse string
+const reverse = function (inString) {
+    let splitString = inString.split("");
+    let reverseArray = splitString.reverse();
+    return reverseArray.join("");
```

Comment:
You can use all of these function together, like this: 
` return inString.split("").reverse().join("");`